### ISSUE TITLE
False positive using intersection types in type annotation

### ISSUE THREAD

**Issue by icanhazstring** [doctrine/coding-standard#162]
Version: 7.0.2

There is a problem using intersection types for type declaration in annotations.
These are valid since PHPStorm 2018.3.

https://blog.jetbrains.com/phpstorm/2018/09/phpstorm-2018-3-eap-183-2635-12/#intersection-types

Given the following code
```
/**
 * @param ContainerInterface&ServiceManager $container
 */
 public function __invoke(ContainerInterface $container) : Fubar {}
```

Expected:
no problem

Actual:
```
FOUND 1 ERROR AFFECTING 1 LINE
----------------------------------------------------------------------
 XX | ERROR | [x] Expected 4 spaces after parameter type; 0 found
```

Using `phpcbf` to fix the issue resulted in this line:
```
* @param ContainerInterface    &ServiceManager $container
```

**Comment by icanhazstring** [comment]
This seems to only affect `@param` annotations.
`@var` annotations for class properties seem fine:

```
/** @var ContainerInterface&ServiceManager */
private $container;
```

**Comment by alcaeus** [comment]
Can you please check which sniff is reporting the error? You can see that by running phpcs with the -s option. Thanks!

**Comment by icanhazstring** [comment]
Hi @alcaeus seems like a squiz sniff is not working properly here :(

```
FOUND 1 ERROR AFFECTING 1 LINE
----------------------------------------------------------------------
 17 | ERROR | [x] Expected 4 spaces after parameter type; 0 found
    |       |     (Squiz.Commenting.FunctionComment.SpacingAfterParamType)
```

**Comment by alcaeus** [comment]
@icanhazstring please check the version of phpcs that you are using and report back on the original issue if it isn't fixed yet. Since this is not an issue with doctrine/coding-standard, I'm closing this issue here.
